plugins/modules/route53_info.py

```diff
@@ -72,7 +72,7 @@
     description:
       - The type of DNS record.
     required: false
-    choices: [ 'A', 'CNAME', 'MX', 'AAAA', 'TXT', 'PTR', 'SRV', 'SPF', 'CAA', 'NS' ]
+    choices: [ 'SOA', 'A', 'TXT', 'NS', 'CNAME', 'MX', 'NAPTR', 'PTR', 'SRV', 'SPF', 'AAAA', 'CAA', 'DS' ]
```

Comment:
If you're updating the choices list, please sort it.  It's tough to follow exactly what changes